### PR TITLE
[NEW COMPONENT]: DigitalElevationMapMsg_CVMat 

### LINKED ISSUE #7: Implement DEM adapter
After release of [dem_msgs](https://github.com/roncapat/dem_msgs), we can implement the corresponding adapter towards OpenCV (like OccupancyGrid).

### PR DESCRIPTION
**Pull request by AntoBongio**
#### Description
Added new adapter from DigitalElevationMap.msg, defined in [dem_msgs](https://github.com/roncapat/dem_msgs), to OpenCV cv::Mat

### CODE REVIEW

File: README.md
Comment (roncapat) [inline README.md]: This should be StampedDigitalElevationImage_CV (?)

File: include/native_adapters/DigitalElevationMapMsg_CvMat.hpp
Comment (roncapat) [inline include/native_adapters/DigitalElevationMapMsg_CvMat.hpp]: Use a different header guard

**Review by roncapat — CHANGES_REQUESTED** [review]
Small fixes :)

**Comment by roncapat** [comment]
CI fails because of the new dependency, I think I will need to change the external repos VCS gist file. Will check.

File: README.md
Comment (AntoBongio) [reply README.md]: Changed with StampedDigitalElevationMap_CV

File: include/native_adapters/DigitalElevationMapMsg_CvMat.hpp
Comment (AntoBongio) [reply include/native_adapters/DigitalElevationMapMsg_CvMat.hpp]: Changed with `NATIVE_ADAPTERS__DIGITALELEVATIONMAP_CVMAT_HPP_`

**Comment by roncapat** [comment]
Will wait for #10 to work before merge

**Review by AntoBongio — COMMENTED** [review]
Checked

File: src/DigitalElevationMapMsg_CvMat.cpp
Comment (roncapat) [inline src/DigitalElevationMapMsg_CvMat.cpp]:
```suggestion
```

**Review by roncapat — CHANGES_REQUESTED** [review]

**Review by roncapat — APPROVED** [review]

**Comment by roncapat** [comment]
@AntoBongio can you fix CI failures?
Have a look at https://github.com/roncapat/ros2_native_adapters/actions/runs/6888535160/job/18737720651

**Comment by roncapat** [comment]
Nice!!

**Pull request merged**
